runtime/verbose/verbose.c

```diff
@@ -1122,6 +1136,19 @@ setVerboseState( J9JavaVM *vm, J9VerboseSettings *verboseOptions, char **errorSt
 		vm->verboseStruct->getCfrExceptionDetails = generateJ9CfrExceptionDetails;
 		vm->verboseStruct->getRtvExceptionDetails = generateJ9RtvExceptionDetails;
 	}
+
+#if (JAVA_SPEC_VERSION >= 11)
+	if (VERBOSE_SETTINGS_CLEAR == verboseOptions->module) {
```

Comment:
I don't understand why this case is needed. It seems the hooks are only registered when VERBOSE_SETTINGS_SET, and registering occurs after parsing all the options, so there is no case when the hooks need to be unregistered. parseVerboseArgumentList() is only called once, and registering the hooks isn't inside a loop.